.github/workflows/ci-latest-release.yml

```diff
@@ -143,3 +165,50 @@ jobs:
         run: |
           STABLE_VERSION=`cat STABLE-RELEASE`
           regctl image copy kubearmor/kubearmor:$STABLE_VERSION kubearmor/kubearmor:stable --digest-tags
+
+      - name: Build & Push KubeArmorController
+        if: steps.match.outputs.tag == 'true'
+        working-directory: ./pkg/KubeArmorController
+        run: PLATFORM=$PLATFORM make docker-buildx TAG=$STABLE_VERSION
```

Comment:
I think we are not populating this environment variable inside this step, which might break things. Is it env variable populated by GitHub itself? If no, let's use `env` block inside this step.

Reply:
yes you are right! i missed it to remove it from here we are handling it in makefile itself, thanks for pointing this out.